stable/redis-ha/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: redis-ha
+home: http://redis.io/
+engine: gotpl
+keywords:
+- redis
+- keyvalue
+- database
+version: 0.2.0
+description: Highly available Redis cluster with multiple sentinels and standbys.
+maintainers:
+  - email: ikaboubi@gmail.com
+    name: KABOUBI
+details:
+  This package provides a highly available Redis cluster with multiple sentinels and standbys.
+  Note the `redis-master` pod is used for bootstrapping only and can be deleted once
+  the cluster is up and running.
```

Comment:
This case can be tested by changing redis master statefulset replicas to 0. You can see that the remain 2 sentinels elect the remain slave after a quorum.
` +odown master mymaster 172.17.0.5 6379 #quorum 2/2`
` +switch-master mymaster 172.17.0.5 6379 172.17.0.8 6379`